.github/ISSUE_TEMPLATE/resourcescheduler.yml

```diff
@@ -0,0 +1,46 @@
+name: Request to schedule events to increase compute
+description: Schedule specific dates/times when your coursework requires higher computing resources
+labels: "support"
+title: Increase compute resources for Course XXX between specific date(s)/time(s)
+assignees:
+  - shaneknapp
+  - sean-morris
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please follow the sections in the template and provide all the required information for us to validate and allocate increased compute for your class.
+  - type: textarea
+    attributes:
+      label: Course Name
+      description: What is the name of the course?
+      value: |
+        <!--Include a link to appropriate webpage of the class if available. If not, please mention class name, instructor and a brief description.-->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Detailed Requirements
+      description: Rationale for requesting increased compute
```

Comment:
Do we need this? Presumable they have >75 students logging in at moment in time

Reply:
fair point...  i will remove this.